tests/network/bindingplugin_passt.go

```diff
@@ -37,7 +37,8 @@ import (
 
 	"kubevirt.io/kubevirt/pkg/libvmi"
 	libvmici "kubevirt.io/kubevirt/pkg/libvmi/cloudinit"
-	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
+	"kubevirt.io/kubevirt/pkg/virt-config/featuregate"
+
```

Comment:
nit: remove new line.

Reply:
The newline separates production from e2e tests packages.